axelrod/fingerprint.py

```diff
@@ -1,60 +1,58 @@
+"""
```

Comment:
ahh.  sorry.  i should have put this top comment here

>because the act of changing position might make it hard to notice any changes, there are notes to myself on any additions/changes (very minor). i'll be using those to comment every line that isn't exactly like the original (minus automatic chnages like removing @staticmethod). and then those can be removed later.

these were those comments i was going to remove.